drivers/console/multi_console.c

```diff
@@ -77,7 +77,11 @@ int console_putc(int c)
 
 	for (console = console_list; console != NULL; console = console->next)
 		if (console->flags & console_state) {
```

Comment:
nit: wouldn't it look cleaner if you just changed this to

    if ((console->flags & console_state) && console->putc)

?